tasks/install-service

```diff
@@ -61,8 +65,11 @@ ExecStart=/usr/bin/podman run --name=cockpit-tasks-%i --hostname=${CONTAINER_HOS
     --volume=\${TEST_SECRETS}/webhook:/run/secrets/webhook:ro \
     --volume=${IMAGE_STORES}:/work/.config/cockpit-dev/image-stores:ro \
     --volume=/etc/npmrc:/etc/npmrc:ro \
+    --volume=%t/podman/podman.sock:/podman.sock:rw \
```

Comment:
Why not use the "normal" location on the inside, and save ourselves the envvar?

Reply:
Heh, I spent about two hours on that. There's no `$XDG_RUNTIME_DIR` in the tasks container, so podman defaults to some `/tmp/podman-1111/whatever` path which doesn't feel very predictable. Plus, if you actually bind-mount it there, it then wants a /work/rundir/podman/... thing. This is by far the easiest and most robust version that I found.

Reply:
I mean `/run/podman/podman.sock`.  I guess that's only the default when you're root, though?

Reply:
But we aren't, we are uid 1111 :grin: